UnityProject/Assets/Scripts/ChemistryComponents/IProvideReagentMix.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Chemistry.Components
+{
+	public interface IProvideReagentMix
```

Comment:
```suggestion
	public interface IReagentMixProvider
```

following naming conventions for interfaces, it would be like that